test/manual/index.html

```diff
@@ -5,7 +5,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <title>mini-css-extract-plugin testcase</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" type="text/css" href="/dist/preloaded1.css" />
+  <link rel="stylesheet" type="text/css" href="/dist/preloaded1.client.css" />
```

Comment:
This gets a mime/type error when testing, but it is the same on master, so I guess that's a potential PR to the main repo.

Reply:
Yeah, it has to do with the `webpack-dev-server` settings. Nothing to worry about though.